doc/developers/contributing.rst

```diff
@@ -469,21 +469,22 @@ documentation with the maths makes it more friendly towards
 users that are just interested in what the feature will do, as
 opposed to how it works "under the hood".
 
-You may also be asked to show your changes when it's built. When you create
-a pull request or make changes in an existing one modifying the docs, CircleCI
-automatically builds them. Thus, you can easily view your changes in the built
-artifacts using the following URL:
-
-``http://scikit-learn.org/circle?{BUILD_NUMBER}``
-
-We attempt to assemble a more precise set of changed files in the
-documentation at:
-
-``http://scikit-learn.org/circle?{BUILD_NUMBER}/_changed.html``
-
-Note: When you visit the details page of the CircleCI tests, you can find your
-BUILD_NUMBER mentioned as 'build #' which is different from your pull request
-number, which is presented as 'pull/#'.
+When you change the documentation in a pull request, CircleCI automatically
+builds it. To view the documentation generated by CircleCI:
+    * navigate to the bottom of your pull request page to see the CI
```

Comment:
I think we want a blank line before this and no indentation

Reply:
Not an rst expert, but it looks fine AFAICS on https://15679-843222-gh.circle-artifacts.com/0/doc/developers/contributing.html#documentation.